### PR TITLE
Setup CODEOWNERS

### PR DESCRIPTION
**Pull request by jwhitlock**
This will automate assigning jwhitlock to code review.

### CODE REVIEW

**Comment by jwhitlock** [comment]
@djmitche does this look like a good start for ``CODEOWNERS``?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Defaults for assigning reviews
+# See https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# and http://code.v.igoro.us/posts/2019/07/codeowners.html
```
Comment (djmitche) [inline .github/CODEOWNERS:3]: haha, I was going to link to that :)

**Review by djmitche — APPROVED** [review]
It does!

**Comment by jwhitlock** [comment]
Thanks @djmitche!

**Pull request merged**